REVIEWING.md

```diff
@@ -0,0 +1,78 @@
+
+## The Reviewer Role
+
+[Instructions](https://contribute.cncf.io/maintainers/github/templates/optional/reviewing/#the-reviewing-role)
+
+⚠️ **Pick the statement that matches the reviewer role for your project**
+
+Only maintainers review pull requests.
+
+OR
+
+The reviewer role is distinct from the maintainer role. Reviewers can approve a pull request but they cannot merge it. A maintainer handles final approval and merging the pull request.
+
+## Values
+
+[Instructions](https://contribute.cncf.io/maintainers/github/templates/optional/reviewing/#values)
+
+
+All reviewers must abide by the [Code of Conduct](CODE_OF_CONDUCT.md) and are also protected by it. A reviewer should not tolerate poor behavior and is encouraged to report any behavior that violates the Code of Conduct. All of our values listed above are distilled from our Code of Conduct.
+
+Below are concrete examples of how it applies to code review specifically:
+
+### Inclusion
+
+Be welcoming and inclusive. You should proactively ensure that the author is successful. While any particular pull request may not ultimately be merged, overall we want people to have a great experience and be willing to contribute again. Answer the questions they didn't know to ask or offer concrete help when they appear stuck.
+
+### Sustainability
+
+Avoid burnout by enforcing healthy boundaries. Here are some examples of how a reviewer is encouraged to act to take care of themselves:
+
+* Authors should meet baseline expectations when submitting a pull request, such as writing tests.
+* If your availability changes, you can step down from a pull request and have someone else assigned.
+* If interactions with an author are not following code of conduct, close the PR and raise it up with your Code of Conduct committee or point of contact. It's not your job to coax people into behaving.
+
+### Trust
+
+Be trustworthy. During a review, your actions both build and help maintain the trust that the community has placed in this project. Below are examples of ways that we build trust:
+
+* **Transparency** - If a pull request won't be merged, clearly say why and close it. If a pull request won't be reviewed for a while, let the author know so they can set expectations and understand why it's blocked.
+* **Integrity** - Put the project's best interests ahead of personal relationships or company affiliations when deciding if a change should be merged.
+* **Stability** - Only merge when then change won't negatively impact project stability. It can be tempting to merge a pull request that doesn't meet our quality standards, for example when the review has been delayed, or because we are trying to deliver new features quickly, but regressions can significantly hurt trust in our project.
+
+## Process
+
+[Instructions](https://contribute.cncf.io/maintainers/github/templates/optional/reviewing/#process)
+
+⚠️ **Define your project's review process**
```

Comment:
We need some links here to example reviewing processes from projects we like.  Unless that's already in the docs somewhere?

Reply:
I don't see it.  Let's hash this out on monday.

Reply:
Ok, now the examples are directly in the process section on the site so that they are easier to find

https://deploy-preview-157--cncf-contribute.netlify.app/maintainers/github/templates/optional/reviewing/#process